libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,83 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprecated from 4.10.0 release of sdk).
         /// </summary>
-        /// <param name="result">Result to be dispalyed as prompts.</param>
+        /// <param name="result">Result to be displayed as prompts.</param>
         /// <param name="cardNoMatchText">No match text.</param>
         /// <returns>IMessageActivity.</returns>
         public static IMessageActivity GetQnAPromptsCard(QueryResult result, string cardNoMatchText)
+        {
+            return GetQnADefaultResponse(result, true, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Get Message Activity Response when User Query is processed by QnAMaker Service.
+        /// </summary>
+        /// <param name="result">Result consisits of References to other answers, also called as Prompts.</param>
```

Comment:
>consisits [](start = 40, length = 9)

Spelling.

Reply:
Updated as per comment.

---
In reply to: [436229585](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r436229585) [](ancestors = 436229585)